OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="advance-retract-container">
+
+  <div class="advance-label">
+    ADVANCE
+    <ion-toggle class="advance-toggle" [disabled]="advanceFlag"></ion-toggle>
```

Comment:
We don't want the "toggle" to be **disable** ,when the "button" is being click. We want the "button" to be **disable**, when the "toggle" is "red".